start_esp/nginx-auto.conf.template

```diff
@@ -64,6 +64,8 @@ http {
   server {
     server_name ${ingress.host};
 
+    ssl_protocols ${ssl_protocols};
```

Comment:
Prefer to not set ssl_protocols as default.

Reply:
Done: changed to None.